src/_includes/components/billboard.njk

```diff
@@ -0,0 +1,49 @@
+{% macro billboard(theme="", title="", venue="") %}
+    {% if theme == "purple" %}
+        {% set titlebar_bg_color = "bg-[#CD89FF]" %}
+        {% set content_bg_color = "bg-[#F3E3FF]" %}
+        {% set title_text_color = "text-[#8E32D2]" %}
+        {% set subtitle_text_color = "text-[#000000A8]" %}
+    {% elif theme == "red" %}
+        {% set titlebar_bg_color = "bg-[#F073B4]" %}
+        {% set content_bg_color = "bg-[#FFE1F1]" %}
+        {% set title_text_color = "text-[#CD1F7A]" %}
+        {% set subtitle_text_color = "text-[#000000A8]" %}
+    {% elif theme == "blue" %}
+        {% set titlebar_bg_color = "bg-[#7291EB]" %}
+        {% set content_bg_color = "bg-[#E9EFFD]" %}
+        {% set title_text_color = "text-[#3D60FF]" %}
+        {% set subtitle_text_color = "text-[#000000A8]" %}
+    {% else %}
+        {% set titlebar_bg_color = "bg-[#8385F2]" %}
+        {% set content_bg_color = "bg-[#F0EEFF]" %}
+        {% set title_text_color = "text-[#7D83F9]" %}
+        {% set subtitle_text_color = "text-[#000000A8]" %}
+    {% endif %}
+
+    <div class="border border-black">
+        <div class="px-4 py-1 border-b border-black {{ titlebar_bg_color }}">
+            <!-- header -->
+            <div class="window-titlebar-dots-reverse">
+                <div class="w-2 h-2 bg-[#D7FF7B] rounded-full"></div>
+                <div class="w-2 h-2 bg-[#D7FF7B] rounded-full"></div>
+                <div class="w-2 h-2 bg-[#D7FF7B] rounded-full"></div>
+            </div>
+            <!--// header -->
+        </div>
+        <div class="p-5 {{ content_bg_color }}" style="display: flex; flex-direction: column; gap: 0.5rem;">
+            <!-- title -->
+            {% if venue == "" %}
+                <p class="text-3xl font-bold text-center {{ title_text_color }}">{{ title }}</p>
+            {% else %}
+                <p class="text-xl {{ title_text_color }}">{{ title }}</p>
+            {% endif %}
+            <!-- // title -->
+
+            <!-- subtitle -->
+            <p class="text-xl {{ subtitle_text_color }} {{ text_align }}">{{ venue }}</p>
+            <!-- //subtitle -->
+            {{ caller() }}
```

Comment:
lets remove this from here!

Reply:
or is there a use case i am missing here?

Reply:
This is designed to make the component extensible. If someone doesn't want to follow the standard title-body format, they can pass custom HTML, which will be rendered within the component window. This approach allows the component to support highly customizable content, making it much more flexible and reusable.

<img width="293" height="95" alt="image" src="https://github.com/user-attachments/assets/e6b76a39-3ff9-4997-8b84-1e05198bcd5d" />

Reply:
Yes, it got a bit overcomplicated. Since we don’t anticipate any immediate use case for it, let’s simplify things for now.

Reply:
removed
```
{{ caller() }}
```